### DISCUSSION TITLE
CAM without SSO name

### DISCUSSION THREAD

**Discussion by TM1Paul** [cubewise-code/tm1py#1054] [Q&A]
Please could I ask for clarification on CAM without SSO user name when connecting to a TM1 server on a TM1Py script.  The base user name is  e.g. CAMID("MSAD:u:22aafed7fca16b42ae6ea64d974c0534") - MSAD is the namespace, but should I be using some or all of this as the name, or the }TM1_DefaultDisplayValue or something else completely?

**Comment by rclapp** [comment]
You should use whatever ID you use to login to tm1 normally. It is likely a user ID or an email.

On Feb 6, 2024 4:14 AM, TM1Paul ***@***.***> wrote:

Please could ask for clarification on CAM without SSO user name when connecting to a TM1 server on a TM1Py script. The base user name is e.g. CAMID("MSAD:u:22aafed7fca16b42ae6ea64d974c0534") - MSAD is the namespace, but should I be using some or all of this as the name, or the }TM1_DefaultDisplayValue or something else completely?
